modules/touch/src/directinputsolver.cpp

```diff
@@ -265,6 +265,70 @@ bool DirectInputSolver::solve(const std::vector<TUIO::TuioCursor>& list,
     return result;
 }
 
+bool DirectInputSolver::solve(const std::vector<TouchInputs>& list, 
+                              const std::vector<SelectedBody>& selectedBodies,
+                              std::vector<double>* parameters, const Camera& camera) 
+{
+    int nFingers = std::min(static_cast<int>(list.size()), 3);
+    _nDof = std::min(nFingers * 2, 6);
+
+    // Parse input data to be used in the LM algorithm
+    std::vector<glm::dvec3> selectedPoints;
+    std::vector<glm::dvec2> screenPoints;
+
+    for (int i = 0; i < nFingers; ++i) {
+        const SelectedBody& sb = selectedBodies.at(i);
+        selectedPoints.push_back(sb.coordinates);
+        screenPoints.emplace_back(
+            2 * (list[i].getLatestInput().x - 0.5),
+            -2 * (list[i].getLatestInput().y - 0.5)
+        );
+
+        // This might be needed when we're directing the touchtable from another screen?
```

Comment:
Generally we do not want to include a lot of commented code. I would suggest making it into an issue to implement functionality to redirect from another screen and add this suggested code there.

Reply:
This comment stems from when I moved out the directinput mean-square-solver to a standalone file. Since then I have switched input modes so I'll remove this comment.